index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://modinate-ui.netlify.app/CSS_files/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link href='https://unpkg.com/boxicons@2.1.1/css/boxicons.min.css' rel='stylesheet'>
+    <link rel="stylesheet" href="./css_files/home_page.css">
+    <title>ModiQuiz</title>
+</head>
+
+<body>
+    <div class="app">
+        <header class='navbar'>
+            <div class="header-left">
+                <a>
+                    <h3>Modi-Quiz</h3>
+                </a>
+            </div>
+            <div class="header-right">
+                <nav>
+                    <ul>
+                        <li>
+                            <a href="https://twitter.com/Chirag__dev">
+                                <i class='bx bxl-twitter social-icons'></i>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://github.com/Hardiegogo/Modinate_UI">
+                                <i class='bx bxl-github social-icons'></i>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://www.linkedin.com/in/chirag-dev/">
+                                <i class='bx bxl-linkedin social-icons'></i>
+                            </a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
+        <main>
+            <div class="hero-section p-24">
+                <div class="hero-left">
+                    <h2>
+                        Want to answer <br> some super interesting Quizes? <br>
+                        Welcome to <br> <span class="primary-text">Modi</span>-Quiz. <br>
+                        <button class="btn btn-primary btn-quiz-hero">Explore quizes</button>
```

Comment:
I think this button can be linked to the component below (categories of quizzes)

Reply:
Got it, thanks Shivani.